src/core/memory.cpp

```diff
@@ -436,6 +447,21 @@ T MemorySystem::Read(const VAddr vaddr) {
         return value;
     }
 
+    // Custom Luma3ds mapping
+    // Is there a more efficient way to do this?
+    if (vaddr & (1 << 31)) {
+        PAddr paddr = (vaddr & ~(1 << 31));
+        if ((paddr & 0xF0000000) == 0x20000000) { // Check FCRAM region
```

Comment:
```suggestion
        if ((paddr & 0xF0000000) == Memory::FCRAM_PADDR) { // Check FCRAM region
```
ditto for below as well